docs/usage/advanced/calico.md

```diff
@@ -29,58 +15,51 @@ So the command of the cluster creation is (when you are at root of the k3d repos
 
 ```bash
 k3d cluster create "${clustername}" \
-  --k3s-arg '--flannel-backend=none@server:*' \
+  --k3s-arg '--flannel-backend=none@server:*' --no-lb \
```

Comment:
Why does the LB have to be disabled?

Reply:
:D Because I use a local loadbalancer.
I've changed the creation step to include load balancer and modified some of the texts to help people know why we are doing some of the stuff in each steps.
thanks for the reviews!